magma_access_gateway_installer/__init__.py

```diff
@@ -112,35 +122,44 @@ def cli_arguments_parser(cli_arguments):
     return cli_options.parse_args(cli_arguments)
 
 
-def validate_args(args, network_interfaces: List[str]):
+def validate_args(args: argparse.Namespace, network_interfaces: List[str]):
+    """Validates operator provided arguments and raises when invalid."""
     validate_static_ip_allocation(args)
     validate_arbitrary_dns(args)
     validate_custom_sgi_and_s1_interfaces(args, network_interfaces)
 
 
-def validate_custom_sgi_and_s1_interfaces(args, network_interfaces: List[str]):
+def validate_custom_sgi_and_s1_interfaces(args: argparse.Namespace, network_interfaces: List[str]):
     if args.sgi or args.s1:
         if not args.sgi or args.sgi not in network_interfaces:
-            raise ValueError("Invalid or empty SGi interface specified! Exiting...")
+            raise ArgumentError("Invalid or empty sgi argument. It must match an interface name.")
```

Comment:
```suggestion
            raise ArgumentError("Invalid or empty --sgi argument. It must match an interface name.")
```

Reply:
Just to be consistent with the other messages (like _--gw-ip-address_ or _--dns_).